mk/check.mk

```diff
@@ -69,17 +69,19 @@ lint: helm-lint golangci-lint shellcheck kube-lint hadolint ginkgo/lint
 
 .PHONY: check
 check: format/common lint ## Dev: Run code checks (go fmt, go vet, ...)
-	# fail if Git working tree is dirty or there are untracked files
-	git diff --quiet || \
-	git ls-files --other --directory --exclude-standard --no-empty-directory | wc -l | read UNTRACKED_FILES; if [ "$$UNTRACKED_FILES" != "0" ]; then false; fi || \
-	test $$(git diff --name-only | wc -l) -eq 0 || \
-	( \
-		echo "The following changes (result of code generators and code checks) have been detected:" && \
-		git --no-pager diff && \
```

Comment:
@michaelbeaumont we're missing `--no-pager` now :(

Reply:
Have you seen CI hanging? I suspect `PAGER` isn't set in CircleCI